tests/test_user_document.py

```diff
@@ -0,0 +1,25 @@
+"""Unit test module for terms of use logic"""
+import json
+from flask_webtest import SessionScope
+from tempfile import NamedTemporaryFile
+from StringIO import StringIO
+
+from tests import TestCase, TEST_USER_ID
+from portal.extensions import db
+from portal.models.user_document import UserDocument
+
+
+class TestUserDocument(TestCase):
+    """User Document tests"""
+
+    def test_post_patient_report(self):
+        #tests whether we can successfully post a patient report -type user doc file
+        service_user = self.add_service_user()
+        self.login(user_id=service_user.id)
+        tmpfile = NamedTemporaryFile(suffix='.pdf')
```

Comment:
Both good points. Added additional file contents check, and moved temp file to within a with statement (just for consistency/cleaner code). For the upload file, I'm currently just deleting it after checking the file contents.